src/Feature/Navigation/client/Header/NavMenu/styles.scss

```diff
@@ -0,0 +1,213 @@
+        text-align: center;
+        @media (max-width: $screen-sm-max){
+          text-align: left;
+          margin-right: 40px; //space for icon
+        }
+      }
+      .dropdown {
+        @include clearfix;
+        position: absolute;
+        left: -999em;
+        z-index: 101;
+        width: 590px;
+        overflow: hidden;
+        display: flex;
+        align-items: stretch;
+        @media (max-width: $screen-sm-max){
+          position: static;
+          width: auto;
+          padding-right: 0;
+          max-height: 0px;
+          transition: max-height .2s;
+        }
+        nav {
+          width: 300px;
+          float:left;
+          padding: 10px 0;
+          @media (max-width: $screen-sm-max){
+            width: auto;
+            float: none;
+          }
+          a {
+            color: $header-section-font-second-color;
+            padding: 5px 25px;
+            display:block;
+          }
+          span.title{
+            cursor: default;
+            padding: 10px 0 5px;
+            position: relative;
+            margin-left: 25px;
+            margin-right: 25px;
+            text-transform: uppercase;
+            display: block;
+            border-style: solid;
+            color: $header-section-font-color;
+            &:after {
+              content: '';
+              margin-bottom: 5px;
+              margin-top: 5px;
+              display: block;
+              position: absolute;
+              left: 0;
+              right: 0;
+              height: 1px;
+              background: $header-section-font-color;
+            }
+          }
+        }
+        aside {
+          float: left;
+          width: 290px;
+          padding: 10px 0;
+          margin: 0;
+          @media (max-width: $screen-sm-max){
+            display: none;
+          }
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          img {
+            display: block;
+            height: auto;
+          }
+        }
+      }
+      .dropdown.drop-open{
+        max-height:100vh;
+      }
+      &:hover .dropdown {
+        left:0px;
+      }
+      //no room to the right
+      &.menu-item {
+        &.right-item:hover {
+          .dropdown{
+            left: auto;
+            right: 0;
+          }
+        }
+        &:hover {
+          background-color: $main-color;
+        }
+        .dropdown {
+          background-color: $main-color;
+          @media (max-width: $screen-sm-max){
+            background: #fff;
+            nav{
+              border-style: solid;
+              border-bottom-width: 10px;
+            }
+            span.title{
+              color: $second-color-dark;
+              border-top-width: 1px;
+              margin-top: 5px;
+            }
+            li{
+              text-transform: none;
+            }
+            a{
+              color:#898b8e;
+            }
+            a:hover {
+              color: #111;
```

Comment:
Use colors variables

Reply:
@evgenii-nikulin

Reply:
We haven't color for that yet, @yuya-yuliya said leave it like that